WordPress/src/main/java/org/wordpress/android/ui/reader/views/ReaderItemDecoration.java

```diff
@@ -0,0 +1,28 @@
+package org.wordpress.android.ui.reader.views;
+
+import android.graphics.Rect;
+import android.support.v7.widget.RecyclerView;
+import android.view.View;
+
+/**
+ * dividers for reader cards
+ */
+public class ReaderItemDecoration extends RecyclerView.ItemDecoration {
```

Comment:
Should this be renamed (and move packages) to something more generic now that it's used in both Reader and Post lists?